Test/git-issues/git-issue-2612B.dfy

```diff
@@ -0,0 +1,8 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+trait T {
+  opaque predicate True() { true }
+}
```

Comment:
You could easily add to this test file something that confirms that the `opaque` keyword is recognized. Something like

``` dafny
method TestOpaque(c: C) {
  assert c.True(); // error: True is opaque and hasn't been revealed
}

method TestRevealed(c: C) {
  reveal c.True(); // test that True can be revealed
  assert c.True(); // now, the assertion goes through
}
```

Reply:
Added.